src/sagemaker/inference_recommender/inference_recommender_mixin.py

```diff
@@ -465,7 +465,7 @@ def _convert_to_endpoint_configurations_json(
             for instance_type in instance_types:
                 parameter_ranges = []
                 for name, param in parameter_range.items():
-                    as_json = param.as_json_range(name)
+                    as_json = {"Name": name, "Values": param.values}
                     as_json["Value"] = as_json.pop("Values")
```

Comment:
I think you don't need this line if you just use "Value" in line above?

Reply:
Do you mean something like:
```
as_json = {"Name": name, "Value": param.values}
```
and remove line 469?